db/migrate/20160715024414_rename_long_column_in_users.rb

```diff
@@ -4,5 +4,6 @@ def up
   end
 
   def down
+  	rename_column :location_tags, :lon, :long
```

Comment:
Why is this necessary? Lalith and I agreed to use the shorter `:lon` for this column? I think we can just leave this out of the PR.